core/cli_entry.py

```diff
@@ -30,10 +31,17 @@ def validate_and_upload_single_entry(directory, force_upload=False):
     logging_start_block(directory)
     logging.debug("validate_and_upload_single_entry:Starting {}".format(directory))
 
+    directory_status = parsing_handler.get_run_status(directory)
+    if directory_status.is_invalid():
```

Comment:
Changed this to check directory status validity before checking if new, because of the case where a run is valid, not new, and we are forcing upload. Previously if you used --force, it would try to upload an run even when it isn't valid.